lib/search_config.rb

```diff
@@ -10,6 +10,8 @@
 require "erb"
 
 class SearchConfig
+  ParseError = BaseParameterParser::ParseError
```

Comment:
why have we done this? And if we do need this would we be better off use inheritance rather than constant assignment (think it would be easier to debug without breaking anything)